services/QuillLMS/client/app/bundles/PremiumHub/components/CanvasInstanceSchoolsSelector.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react'
+
+import { DataTable } from '../../Shared/index';
+
+const CanvasInstanceSchoolsSelector = ({ schools, setSelectedSchoolIds, selectedSchoolIds }) => {
+  const checkRow = (id) => {
```

Comment:
totally nbd but it might be a little cleaner to do:

```
const checkRow = (id) => {
  if (selectedSchoolIds.includes(id)) { return }

  setSelectedSchoolIds([...selectedSchoolIds, id])
}

Reply:
I like the guard clause more.  I wish [eslint](https://github.com/eslint/eslint/issues/10858) had a linter for guard clauses like rubocop.  Maybe I will write one at some point.

Reply:
@emilia-friedberg Good suggestion.  I like the guard clause more.  I wish [eslint](https://github.com/eslint/eslint/issues/10858) had a linter for guard clauses like rubocop.  Maybe I will write one at some point.